nativelib/src/main/resources/scala-native/scala-native.properties

```
@@ -0,0 +1,13 @@
+# output via debugging
+project.groupId = org.scala-native
+project.artifactId = nativelib
+# setup define based on GC selected
+project.gcProject = true
+
+# CG define
+preprocessor.defines = NDEBUG
+
+# path to vendored libunwind
+compile.include.path = platform, posix, libunwind
```

how about storing relative unix path?
```suggestion
compile.include.path = platform/posix/libunwind
``` 
We can then simply replace all `/` to `java.io.File.separator`

---

I could do that and then we could just have.
```
compile.include.paths = platform/posix/libunwind, gc
```

That could simplify things on the implementation side.